src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -20,19 +22,19 @@
             <argument>null</argument><!-- var_dumper.cli_dumper when debug.dump_destination is set -->
         </service>
 
-        <service id="debug.dump_listener" class="Symfony\Component\HttpKernel\EventListener\DumpListener">
+        <service id="debug.dump_listener" class="Symfony\Component\HttpKernel\EventListener\DumpListener" public="true">
```

Comment:
it would be great to be able to change this service to become private in 4.0 (as event listeners can now be private).
@nicolas-grekas I thnk a necessary feature for 3.4 would be a way to mark a service as "wanna-be-private", which would trigger a deprecation warning when getting it from the container through `get`, but not when injecting it inside another service (unlike deprecated services). Such feature would allow any bundle out there to provide gradual migration to private services (warning people if they try to access internal services that will become private)